src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+
+Item {
```

Comment:
Can probably get rid of this `Item {}` (though I know a few of the settings sub-pages follow this pattern)

Reply:
I remember trying to avoid using this at the start and was running into either some weird qml warning or layout issues and figured it was some hacky workaround, but let me try again!